front/app/containers/Admin/projects/project/index.test.tsx

```diff
@@ -41,7 +38,7 @@ const getRouterProps = (projectId, tabName?: string) =>
 
 const children = () => <div />;
 
-describe('<AdminProjectEdition />', () => {
+describe.skip('<AdminProjectEdition />', () => {
```

Comment:
This is the only test suite that didn't work with minor changes, I think due to the recent refactor of the component. @brentguf can you take a look at this one as part of looking into that component?

Reply:
@lilkraftwerk What's the error?

Reply:
Something to do with the component wrapper in the tests, along the lines of `props is meant to be called on at least 1 node, but 0 were found`. @brentguf refactored this component recently to make it work with the new router changes so I think it's likely due to the changed structure of the component

Reply:
Alright 👍 It's an enzyme test and we are trying to get rid of these. Would it be feasible to re-create it in rtl - perhaps in a separate PR?

Reply:
Shall I look at that after merging my fixes to the project component? Logged [here](https://citizenlab.atlassian.net/browse/CL-867).

@ Iva, as I'm still a newbie about RTL tests, would you mind pairing on this next week?

Reply:
Sure, happy to, Brent :)